de-lib/src/main/java/org/iplantc/de/client/services/impl/DiskResourceServiceFacadeImpl.java

```diff
@@ -1068,6 +1068,11 @@ public void requestHTPathlistFile(HTPathListRequest request, DECallback<File> ca
                 address.append("&info-type=" + infotype);
             }
         }
+
+        if(request.getPathListType() != null) {
+            address.append("&path-list-type=" + request.getPathListType());
```

Comment:
⚠️ Note: I just updated this parameter to `path-list-info-type` in cyverse-de/data-info#16

Reply:
Ahh cool. Let me update that in UI as well.